membership-common/src/main/scala/com/gu/memsub/subsv2/reads/ChargeListReads.scala

```diff
@@ -152,16 +139,9 @@ object ChargeListReads {
     }.withTrace("readPaidChargeList")
   }
 
-  implicit def readFreeChargeList: ChargeListReads[FreeChargeList] = new ChargeListReads[FreeChargeList] {
-    def read(cat: PlanChargeMap, charges: List[ZuoraCharge]): ValidationNel[String, FreeChargeList] = {
-      readFreeCharge[Benefit](readAnyProduct).read(cat, charges).map(identity[FreeChargeList])
-    }
-  }
-
   implicit def readChargeList: ChargeListReads[ChargeList] = new ChargeListReads[ChargeList] {
     override def read(cat: PlanChargeMap, charges: List[ZuoraCharge]) = {
-      readPaidChargeList.read(cat, charges) orElse2
-        readFreeChargeList.read(cat, charges)
+      readPaidChargeList.read(cat, charges).map(identity[ChargeList])
```

Comment:
need to map with identity because functions are (co)variant but a ChargeListRead is not.  This means 
`ChargeListReads[PaidChargeList]` is not a subclass of `ChargeListReads[ChargeList]`

Whether it should be or not was not something I wanted to delve in to, since this function will disappear in the next PR.